lib/shopify-cli/commands/generate.rb

```diff
@@ -23,8 +23,9 @@ def self.extended_help
           {{bold:Subcommands:}}
 
             {{cyan:page}}: Generate a new page in your app with the specified page name.
-              Usage: {{command:#{ShopifyCli::TOOL_NAME} generate page <pagename>}}
-
+              Usage: {{command:#{ShopifyCli::TOOL_NAME} generate page <pagename>}} or
+                     {{command:#{ShopifyCli::TOOL_NAME} generate page <pagename> --type=TYPE}}
+              Types:
               {{cyan:empty-state}}: generate a new page with an empty state
```

Comment:
The explanatory text after the colon should be in sentence case — capitalized with punctuation at the end!